e2e/Configuration.e2e.js

```diff
@@ -0,0 +1,6 @@
+describe('Configuration screen', () => {
```

Comment:
Maybe you should have a beforeEach here too. Do the tests run always in the same order?

Reply:
I just checked and detox relaunches the app before starting each e2e file, so we should be covered. Would you prefer to add it anyways just for clarity or it's ok?